pkg/scheduler/api/types.go

```diff
@@ -116,6 +116,9 @@ type LessFn func(interface{}, interface{}) bool
 // CompareFn is the func declaration used by sort or priority queue.
 type CompareFn func(interface{}, interface{}) int
 
+// CompareFn is the func declaration used by sort or priority victims.
```

Comment:
VictimCompareFn maybe better
```suggestion
// VictimCompareFn is the func declaration used by sort or priority victims.
```